src/beacon/app/routes/analytics.py

```
@@ -0,0 +1,481 @@
+    *,
+    db: Session = Depends(get_db),
+    days: int = Query(7, ge=1, le=30)
+):
+    """
+    Get analysis grouped by region
+    """
+    end_date = datetime.utcnow()
+    start_date = end_date - timedelta(days=days)
+    
+    # Get unique regions
+    regions = db.exec(select(Site.region).distinct().where(Site.region != None)).all()
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)